### ISSUE TITLE
Can't sync trophies

### ISSUE THREAD

**Issue by ElruXeno** [bucanero/apollo-ps3#104]
What i did: 
- installed CWF 4.89, webman and latest Apollo.
- Synced trophies and got a trophy set
- Unlocked a trophy with Apollo (selected, pressed apply)
- Rebuilded a database
- Trophy appered in Offline folder
- After i successfuly sync it still unlocked in Offline but no changes in Online trophy folder.

I reinstalled whole system just in case and repeated the steps - no changes. For experiment i tried a game which already had data and another game, where no trophies unlocked. In the second case i even got an error 80030C08 while syncing and after that trophy is gone (still showed in Apollo as unlocked). I rebuilded database many times but still can't sync any trophy for some reason. Just for  the clarification. i'm using the same console and account as 10 years ago, nothing is spoofed (i checked). 

While searching for the solution i found a guy with the same problem. I hope somebody can help us.

**Comment by bucanero** [comment]
"Trophies" code is based on the PS3TrophyIsGood PC tool, if you have issues with Apollo you can try the PC tool and compare. 

This was originally tested with 4.84, and it's possible that server changes in 4.89 turned this code incompatible.
It was never meant for cheating and syncing, so if you want to keep your account data in shape I suggest to earn trophies by playing and avoid tampering the files.

For reference, there's a "trophy plugin" floating around that can mass-unlock trophies for a game. Again, I'm not promoting this, since it's cheating and all the trophies will have the same timestamp (and could get you a ban)

**Comment by Masamune3210** [comment]
Wondering passerby with a heads up, trophy modding is one of the few things sony still pays attention to on the ps3, and if you value your account not getting attacked its probably a good idea to not mess with modifying them too much, especially unlocking entire sets at once

**Comment by ElruXeno** [comment]
Yes, i'm fully aware that it might be dangerous and i was willed to take a risk. I alro tried  PS3TrophyIsGood PC but getting an error "Open failed: Not a valid TROPTRNS.DAT". Also tried before and after reinstalling the system, just to be sure that database is fine and it's not on my end. 

However, i'm not gonna use "trophy plugin" because it's too risky and obvious.
